contrib/ay-openqa-worker-leap.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <general>
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+    </mode>
+  </general>
+  <networking>
+    <!-- For multiple network interfaces by default the last is configured, prefer the first -->
+    <interfaces config:type="list">
+      <interface>
+        <bootproto>dhcp</bootproto>
+        <name>eth0</name>
+        <startmode>auto</startmode>
+      </interface>
+    </interfaces>
+    <keep_install_network config:type="boolean">true</keep_install_network>
+  </networking>
+  <software>
+    <!-- It seems unfortunately we need to explicitly specify a product, would have preferred a more generic config -->
+    <products config:type="list">
+      <product>Leap</product>
```

Comment:
Can we [use a condition here?](https://doc.opensuse.org/projects/autoyast/#erb-os-release-helper)
```suggestion
      <% if os_release[:id] == 'sle' %>
      <product>SLES</product>
      <% else %>
      <product>Leap</product>
      <% end %>
```